HW02/Aliaksandr_Korzun/aliaksandr_korzun_hw_02_t_01.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+logs = 'logfile.txt'
```

Comment:
We can use constant here
```suggestion
LOG_FILE = 'logfile.txt'
```

Reply:
ok, fixed